contracts/interfaces/handlers/IBosonAccountHandler.sol

```diff
@@ -353,6 +370,17 @@ interface IBosonAccountHandler is IBosonAccountEvents {
         BosonTypes.AuthToken calldata _associatedAuthToken
     ) external view returns (bool exists, BosonTypes.Seller memory seller, BosonTypes.AuthToken memory authToken);
 
+    /**
+     * @notice Gets the details about a seller's collections.
+     *
+     * @param _sellerId - the id of the seller to check
+     * @return defaultVoucherAddress - the address of the default voucher contract for the seller
+     * @return additionalCollections - an array of additional collections that the seller has created
+     */
+    function getSellersCollections(
+        uint256 _sellerId
+    ) external view returns (address defaultVoucherAddress, BosonTypes.Collection[] memory additionalCollections);
```

Comment:
Do we worry about the Collection[] being too big ? Do we need something like a `getCollection(CollectionID)` or something ?

Reply:
even if this is a good idea, i wouldn't block on it ...

Reply:
I also think is a good idea, nice catch. 
I create an issue so this doesn't get loss 
https://github.com/bosonprotocol/boson-protocol-contracts/issues/709